schematics/dashboard/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.__styleext__

```diff
@@ -0,0 +1,21 @@
+.grid-container {
+  margin: 20px;
+}
+
+mat-card {
```

Comment:
Kind of nit-picky, but since this could set the pattern for a project I'll get into it:

I think each of these styles that currently target a `mat` element or attribute should be a css class instead. While those selectors are fine in this skeleton, I don't want to give people the impression that this is the way they should _always_ style custom components. Introducing a separate css class signals the intent that you want to style a _specific_ instance of a component rather than _all_ instances of that component.